components/server/src/auth/resource-access.ts

```diff
@@ -237,15 +237,19 @@ export class OwnerResourceGuard implements ResourceAccessGuard {
             case "team":
                 switch (operation) {
                     case "create":
-                        // Anyone can create a new team.
-                        return true;
+                        // Anyone who's not owned by another org can create orgs.
+                        return !resource.members.some((m) => m.userId === this.userId && m.isOwnedByOrganization);
```

Comment:
@svenefftinge, how is "not owned by another org" translated to a predicate based on `resource.members`? Isn't the members array filled with a single team/org typically? And shouldn't that check depend on `!this.user.organizationId`?